app/partials/data-export-button.cjsx

```diff
@@ -14,15 +13,22 @@ module.exports = React.createClass
   getInitialState: ->
     exportRequested: false
     exportError: null
+    mostRecent: null
+
+  componentDidMount: ->
+    @exportGet()
 
   exportGet: ->
-    @_exportsGet or= @props.project.get(@props.exportType).catch( -> [])
+    @props.project.get(@props.exportType)
+      .then ([exported]) =>
+        @setState mostRecent: exported
+      .catch (error) =>
+        @setState exportError: error
```

Comment:
Sorry one last thing, could this only set the error state if the error status is not a 404? We don't need to report 404s to the user.